internal/cmd/options.go

```diff
@@ -32,6 +34,10 @@ func NewLeaderElectionOptions() (LeaderElectionOptions, error) {
 		}
 		leaderOpts.LeaseDuration = &v
 	}
+	if leaderOpts.LeaseDuration == nil {
```

Comment:
Do we need something similar with the renew deadline and retry period? :thinking:

Reply:
Not according to https://github.com/rancher/fleet/pull/3463#discussion_r2028648698

Reply:
Or maybe we do. I misread that comment as LeaseDuration, when it says Duration